lib/rules/no-debug.ts

```diff
@@ -8,12 +8,13 @@ import {
   isObjectPattern,
   isProperty,
 } from '../node-utils';
+import { DEBUG_UTILS } from '../utils';
 import { createTestingLibraryRule } from '../create-testing-library-rule';
 import { ASTUtils, TSESTree } from '@typescript-eslint/experimental-utils';
 
 export const RULE_NAME = 'no-debug';
 export type MessageIds = 'noDebug';
-type Options = [];
+type Options = [{ utilNames: Array<typeof DEBUG_UTILS[number]> }];
```

Comment:
`utilNames` is optional

```suggestion
type Options = [{ utilNames?: Array<typeof DEBUG_UTILS[number]> }];
```

Reply:
Oh yeah, nice catch!